frontend/packages/console-app/src/components/quick-starts/catalog/QuickStartTileHeader.tsx

```diff
@@ -17,25 +18,35 @@ const statusColorMap = {
   [QuickStartStatus.NOT_STARTED]: 'grey',
 };
 
-const QuickStartTileHeader: React.FC<QuickStartTileHeaderProps> = ({ status, duration, name }) => (
-  <div className="co-quick-start-tile-header">
-    <Title headingLevel="h3">{name}</Title>
-    <div className="co-quick-start-tile-header__status">
-      {status !== QuickStartStatus.NOT_STARTED && (
-        <Label
-          className="co-quick-start-tile-header--rightmargin"
-          variant="outline"
-          color={statusColorMap[status]}
-          icon={<StatusIcon status={status} />}
-        >
-          {status}
+const QuickStartTileHeader: React.FC<QuickStartTileHeaderProps> = ({ status, duration, name }) => {
+  const { t } = useTranslation();
+
+  const statusLocaleMap = {
+    [QuickStartStatus.COMPLETE]: t('quickstart~Complete'),
+    [QuickStartStatus.IN_PROGRESS]: t('quickstart~In Progress'),
+    [QuickStartStatus.NOT_STARTED]: t('quickstart~Not started'),
+  };
+
+  return (
+    <div className="co-quick-start-tile-header">
+      <Title headingLevel="h3">{name}</Title>
+      <div className="co-quick-start-tile-header__status">
+        {status !== QuickStartStatus.NOT_STARTED && (
+          <Label
+            className="co-quick-start-tile-header--rightmargin"
+            variant="outline"
+            color={statusColorMap[status]}
+            icon={<StatusIcon status={status} />}
+          >
+            {t(statusLocaleMap[status])}
```

Comment:
Do we need to wrap in `t` again as `statusLocaleMap[status]` will evaluate to  `t('quickstart~Complete')` ?

Reply:
Had forgotten to remove it. Updated.